README.md

```diff
@@ -1,6 +1,7 @@
 qBittorrent - A BitTorrent client in Qt
 ------------------------------------------
 
+![GitHub Actions CI](https://github.com/qbittorrent/qBittorrent/workflows/GitHub%20Actions%20CI/badge.svg)
```

Comment:
Please make it clickable like the other one. Also I would suggest moving to below Appveyor.

Reply:
Shouldn't you add "Status" to badge name?

Reply:
@glassez @Chocobo1 

Fixed, it's clickable now. Try it out here: https://github.com/FranciscoPombal/qBittorrent/tree/status_badge

![image](https://user-images.githubusercontent.com/17580742/94319943-a3ac5400-ff83-11ea-8fa1-8b2a0a55a256.png)

Reply:
In case you missed it:
>Also I would suggest moving to below Appveyor.

Reply:
@Chocobo1 yeah, sorry about that, fixed.

Reply:
In case you still missed it:
>Shouldn't you add "Status" to badge name?

Reply:
@glassez oh, you meant adding the literal string "status"? Why? The status is already shown, e.g. "passing" and it's clear from the context. Adding "status" is redundant and bloats the labels. The others don't even have it. Though the others have other problems; they should have some indication of what service they refer to about (granted, at least the Appveyor one shows the logo), it's not that helpful if they are all named "build".

Reply:
I mean the following:
>[TravisCI **Status**]
[AppVeyor **Status**]
[Coverity **Status**]

Reply:
@glassez Fixed. My bad... I did not realize before that there was a "Status" that was there but not being displayed.

Reply:
>Though the others have other problems; they should have some indication of what service they refer to about (granted, at least the Appveyor one shows the logo), it's not that helpful if they are all named "build".

👍